$OpenBSD: patch-src_mumble_Audio_cpp,v 1.2 2020/02/25 05:14:52 rsadowski Exp $

Fix loading of CELT plugins; use celt from our tree

Index: src/mumble/Audio.cpp
--- src/mumble/Audio.cpp.orig
+++ src/mumble/Audio.cpp
@@ -53,7 +53,7 @@ void CodecInit::initialize() {
 		delete codec;
 	}
 #else
-	codec = new CELTCodec070(QLatin1String("0.7.0"));
+	codec = new CELTCodec070(QLatin1String("07"));
 	if (codec->isValid()) {
 		codec->report();
 		g.qmCodecs.insert(codec->bitstreamVersion(), codec);
@@ -68,19 +68,12 @@ void CodecInit::initialize() {
 		}
 	}
 
-	codec = new CELTCodec011(QLatin1String("0.11.0"));
+	codec = new CELTCodec011(QLatin1String("11"));
 	if (codec->isValid()) {
 		codec->report();
 		g.qmCodecs.insert(codec->bitstreamVersion(), codec);
 	} else {
 		delete codec;
-		codec = new CELTCodec011(QLatin1String("2.0.0"));
-		if (codec->isValid()) {
-			codec->report();
-			g.qmCodecs.insert(codec->bitstreamVersion(), codec);
-		} else {
-			delete codec;
-		}
 	}
 #endif
 }
